seqr/views/apis/saved_variant_api.py

```diff
@@ -62,12 +61,12 @@ def create_saved_variant_handler(request):
     saved_variant_guids = []
     for single_variant_json in variants_json:
         try:
-            parsed_variant_json = _get_parsed_variant_args(single_variant_json, family)
+            create_json, update_json = parse_saved_variant_json(single_variant_json, family)
```

Comment:
when in the seqr user flow are saved variants created?

Reply:
The first time a user adds a tag or a note to a variant